Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[workspace]
+members = [
+    "crates/dtrace-sys",
+]
```

Comment:
So I've researched how to specify the `dtrace-sys` only works on Mac. There is no way to specify at the crate level "I only work on a Mac" - there is only target-specific dependencies.

But one can specify which packages build by default when one runs `cargo build` in the virtual workspace: https://doc.rust-lang.org/cargo/reference/workspaces.html

..which is what I suggest doing here (and in our code once we test it here, hehe)

```suggestion
[workspace]
members = [
    "crates/dtrace-sys",
]

# Specify a subset of member crates that compile on all supported architectures
default-members = [
]
```